compiler/env/TRPersistentMemory.cpp

```diff
@@ -99,7 +99,9 @@ TR_PersistentMemory::printMemStats()
    fprintf(stderr, "TR_PersistentMemory Stats:\n");
    for (uint32_t i = 0; i < TR_MemoryBase::NumObjectTypes; i++)
       {
-      fprintf(stderr, "\t_totalPersistentAllocations[%s]=%lu\n", objectName[i], (unsigned long)_totalPersistentAllocations[i]);
+      unsigned long allocSize = (unsigned long)_totalPersistentAllocations[i];
```

Comment:
Why do you cast rather than use the declared type of the field (`size_t`)?

The convention we're aspiring to in the compiler is to not use types like `unsigned long` but the more size-explicit types like `uint64_t`.

Reply:
That's just a copy from previous source code to print formatted value, here is **%lu**